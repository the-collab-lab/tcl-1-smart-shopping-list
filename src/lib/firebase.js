// NOTE: import only the Firebase modules that you need in your app... except
// for the second line, which makes both the linter and react-firebase happy
import firebase from 'firebase/compat/app';
import 'firebase/compat/firestore';

// Initalize Firebase.
const config = {
    apiKey: "AIzaSyDvVB6SIAhXqa3chcJgaJi_XGkd1W3xIhI",
    authDomain: "tcl-1-smart-shopping-list.firebaseapp.com",
    projectId: "tcl-1-smart-shopping-list",
    storageBucket: "tcl-1-smart-shopping-list.appspot.com",
    messagingSenderId: "490626563800",
    appId: "1:490626563800:web:1c6c138d68d09a6f8de1ad"
};

let fb = firebase.initializeApp(config);

export { fb };
