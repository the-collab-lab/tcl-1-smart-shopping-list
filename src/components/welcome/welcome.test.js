import React from 'react';
import ReactDOM from 'react-dom';
import { BrowserRouter } from 'react-router-dom';
import TestRenderer from 'react-test-renderer';
import { fb as firebase } from '../../lib/firebase.js';
import { FirestoreProvider } from 'react-firestore';
import Welcome from './index';
it('renders without crashing', () => {
  const div = document.createElement('div');
  ReactDOM.render(
      <BrowserRouter>
        <Welcome />
      </BrowserRouter>,
    div
  );
  ReactDOM.unmountComponentAtNode(div);
});

it('snapshot on load', () => {
  const tree = TestRenderer.create(
      <BrowserRouter>
        <Welcome />
      </BrowserRouter>
  );
  expect(tree.toJSON()).toMatchSnapshot();
});
