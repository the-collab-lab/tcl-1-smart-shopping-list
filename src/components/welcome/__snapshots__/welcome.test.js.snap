// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`snapshot on load 1`] = `
<footer
  className="footer css-rxnh45"
>
  <section>
    <p>
      No purchases just yet.
    </p>
    <a
      href="/add-item"
      onClick={[Function]}
    >
      Add Item
    </a>
  </section>
</footer>
`;
